### PR TITLE
New version: glzr-io.glazewm version 3.6.0

### PR DESCRIPTION
**Pull request by schembriaiden**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/191727)

### CODE REVIEW

File: manifests/g/glzr-io/glazewm/3.6.0/glzr-io.glazewm.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+InstallerType: burn
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{0F9E07AA-23F7-4975-9F97-71F0F758D5B7}'
+ReleaseDate: 2024-11-14
+Installers:
```
Comment (stephengillie) [inline manifests/g/glzr-io/glazewm/3.6.0/glzr-io.glazewm.installer.yaml:15]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @vikingnope,

image
image

**Comment by MoaidHathot** [comment]
@vikingnope, thanks for the update!
GlazeWM has an auto-releaser, I think we should invest in fixing it so we won't need to keep manually updating the manifests. The manual manifest updates are somehow causing `winget update` to not work.

**Comment by vikingnope** [comment]
> @vikingnope, thanks for the update! GlazeWM has an auto-releaser, I think we should invest in fixing it so we won't need to keep manually updating the manifests. The manual manifest updates are somehow causing `winget update` to not work.

Ye I had worked on the auto releaser, don't know why it's not working because it should be working normally

**Review by ItzLevvie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**